.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 Communication Service/Software Laboratory, National Chiao Tung University (free5gc.org)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Main workflow
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.4'
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run: go test -v ./...
```

Comment:
Should not this be deleted given that you already have a `build` and `unit-tests` actions below?

Reply:
I could swear I had removed those, but seems not! I deleted that duplicate section.